mirrord/config/src/ci.rs

```diff
@@ -0,0 +1,54 @@
+use std::path::PathBuf;
+
+use mirrord_analytics::CollectAnalytics;
+use mirrord_config_derive::MirrordConfig;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use crate::config::source::MirrordConfigSource;
+
+/// Configuration for mirrord for CI.
+///
+/// Logs from the app are sent to `/dev/null` when using `mirrord ci start`. You can pipe them
+/// to a file by setting one of the `stdio` file paths (note that the file doesn't have to exist,
+/// mirrord will take care of creating it).
+///
+/// ```json
+/// {
+///   "ci": {
+///     "stdout_file": "/tmp/mirrord-ci-logs/stdout",
+///     "stdin_file": "/tmp/mirrord-ci-logs/stdin",
+///     "stderr_file": "/tmp/mirrord-ci-logs/stderr"
+///   }
+/// }
+/// ```
+#[derive(MirrordConfig, Clone, Debug, Serialize, Deserialize, PartialEq, Default)]
+#[config(map_to = "CiFileConfig", derive = "JsonSchema")]
+#[cfg_attr(test, config(derive = "PartialEq, Eq"))]
+pub struct CiConfig {
+    /// ### ci.stdout_file {#ci-stdout_file}
+    ///
+    /// Pipe `stdout` to the file specified here.
+    pub stdout_file: Option<PathBuf>,
+
+    /// ### ci.stdin_file {#ci-stdin_file}
+    ///
+    /// Pipe `stdin` to the file specified here.
+    pub stdin_file: Option<PathBuf>,
```

Comment:
We don't need a stdin file right?

Reply:
https://github.com/metalbear-co/mirrord/pull/3669/commits/2168ae199bb9f07ed7d0be5254a60fc17aa2ee2c